pkg/controller/util/status.go

```diff
@@ -12,18 +12,18 @@ import (
 // These constants represent the overall Phase as used by .Status.Phase
 var (
 	// PhaseIgnored is used when a resource is ignored
-	PhaseIgnored string = "Ignored"
+	PhaseIgnored = "Ignored"
```

Comment:
These changes should at least be a separate commit.

Reply:
Everything should be in proper commits. :) That's coming later.